firmware/boron-ins-fsm/src/flashAddresses.h

```diff
@@ -38,6 +38,10 @@
 #define ADDR_INITIALIZE_STATE0_OCCUPANT_DETECTION_TIMER_FLAG 29  // uint16_2 = 2 bytes
 #define ADDR_STATE0_OCCUPANT_DETECTION_TIMER                 31  // uint32_t = 4 bytes
 
-// next available address is 31 + 4 = 35
+// bluetooth antenna type
+#define ADDR_INITIALIZE_BLE_ANTENNA_TYPE_FLAG 35  // uint16_t = 2 byte
+#define ADDR_BLE_ANTENNA_TYPE                 37  // uint8_t = 1 byte
```

Comment:
I'd also change these names to make it clearer that true means external.

And the second value is now a boolean, right? So the comment can be updated?

Reply:
@johnny-brave , do you envision there being further BLE antenna types beyond 'internal' and 'external'? If not, this should probably be updated to something like `ADDR_USING_EXTERNAL_BLE_ANTENNA` or something